src/components/forms/AssignTeamTableCell/index.tsx

```diff
@@ -128,14 +134,31 @@ export { AssignTeamTableCell };
 const mapStateToProps = (state: Partial<Store>, ownProps: any): AssignTeamCellProps => {
   const organizationsById = getOrganizationsById(state);
   const assignmentsArray = getAssignmentsArrayByPlanId(state, ownProps.planId);
-  const assignments = assignmentsArray.filter(
-    (a: Assignment) => a.jurisdiction === ownProps.jurisdictionId
-  );
+  const idsToCheckAssignments =
+    ownProps.parentIds && ownProps.parentIds.length
+      ? [ownProps.jurisdictionId, ...ownProps.parentIds]
+      : [ownProps.jurisdictionId];
+  const parentAssignments: string[] = [];
+  const uniqueOptionsIds: string[] = [];
+  const assignments: Assignment[] = [];
+  assignmentsArray.forEach((a: Assignment) => {
+    if (idsToCheckAssignments.includes(a.jurisdiction)) {
+      if (a.jurisdiction !== ownProps.jurisdictionId) {
+        parentAssignments.push(a.organization);
+      }
+      if (!uniqueOptionsIds.includes(a.organization)) {
+        uniqueOptionsIds.push(a.organization);
+        assignments.push(a);
+      }
+    }
+  });
```

Comment:
what is this code doing?

Reply:
- It combines the 'higher' jurisdiction ids with the current jurisdiction id.
- Gets team assignments for the Jurisdiction together  with 'parent' assignments 
- Gets 'parentAssignments' - important to avoid double saving in the API